DependencyInjection/Configuration.php

```diff
@@ -208,4 +210,56 @@ private function addVisitorsSection(NodeBuilder $builder)
             ->end()
         ;
     }
+
+    private function addContextSection(NodeBuilder $builder)
+    {
+        $builder
+            ->arrayNode('context')
```

Comment:
If you like you can also create `default_context`, `default_serialization_context` and `default_serialization_context`. Where is possible to define only  `default_context`. Up to you